### DISCUSSION TITLE
Give admin control of entry right sidebar widgets that plugins add

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/cms#8597] [Ideas]
Plugins can append content to right sidebar in entry page. Most famous use case of this is seomatic plugin that displays seo preview there.

Recently i installed other plugin that also takes up this space - "element map". My problem is that i rarely use "element map" functionality and use seomatic seo preview much more often. But "element map" placed its widget _before_ seomatic widget, so each time i have to scroll down right sidebar to see seomatic seo preview.

What i propose is giving admin some kind of ability to sort these widgets. Or even expand and hide them with "accordion" functionality, with state of widget being saved to cookie (expanded or no).

**Comment by jsunsawyer** [comment]
This would be great. It's especially strange that `Notes about your changes` is pushed below everything that plugins add, since it's part of Craft.
